erpnext/config/buying.py

```diff
@@ -228,29 +228,5 @@ def get_data():
 				}
 			]
 		},
-		{
-			"label": _("Help"),
-			"items": [
-				{
-					"type": "help",
-					"label": _("Customer and Supplier"),
-					"youtube_id": "anoGi_RpQ20"
-				},
-				{
-					"type": "help",
```

Comment:
Why you have removed help articles? update it if any.

Reply:
Was asked to remove them. They are links to some old (outdated) YouTube videos.